test/content/support.js

```diff
@@ -241,4 +241,17 @@ function resetDB() {
 	}).then(function() {
 		return Zotero.Schema.schemaUpdatePromise;
 	});
-}
\ No newline at end of file
+}
+
+
+/**
+ * Imports an attachment from a test file.
+ * @param {string} filename - The filename to import (in data directory)
+ * @returns {Promise} as returned by Zotero.Attachments.importFromFile
```

Comment:
Should update this with what it's actually returning (which is now `{Promise<Zotero.Item>}`. Don't need the rest of the line. Also we use `@return`, not `@returns`.